README.md

```diff
@@ -137,6 +137,7 @@ OpenTelemetry can collect tracing data automatically using plugins. Vendors/User
 - [@opentelemetry/plugin-redis][otel-plugin-redis]
 - [@opentelemetry/plugin-ioredis][otel-plugin-ioredis]
 - [@opentelemetry/plugin-dns][otel-plugin-dns] - By default, this plugin is not loaded [#612](https://github.com/open-telemetry/opentelemetry-js/issues/612)
+- [@opentelemetry/plugin-express][otel-plugin-express]
```

Comment:
I think we should add a note here: "By default, this plugin is not loaded"

Reply:
Is it ? I thought we aimed to emable most of the plugins by default ?

Reply:
I've made the change since we will discuss that with #783

Reply:
The reason I mentioned this is because express is not part of the default list of loaded plugins: https://github.com/open-telemetry/opentelemetry-js/blob/master/packages/opentelemetry-node/src/config.ts#L29

I think we should update the list and then update the README or both at the same time.